metrics/network/remote_network/remote-networking-test-common.sh

```diff
@@ -59,11 +81,11 @@ function setup_swarm {
 	$DOCKER_EXE service create \
 		--name "$first_service" --replicas "$number_of_replicas" \
 		--constraint 'node.labels.machine == machine1' \
-		--publish "$port_first_service" "$network_image" sh
+		--publish "$port_first_service" "$network_image" sh -c "sleep 60"
```

Comment:
if these are meant to effectively 'run forever', and we kill them off/shut them down by hand later, maybe it is better to use something like `fail -f /dev/null` as the payload, rather than the `sleep 60` - having sleeps in the tests always makes us a little nervous for future-proofness.